lib/user_agent/browsers/pod_2_watch.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+class UserAgent
+  module Browsers
+    ##
+    # This class detects and handles the Pod2Watch browser on Apple Watch
+    class Pod2Watch < Base
+      # The Pod2Watch browser
+      BROWSER = 'Pod2Watch'
+
+      ##
+      # Check if this class is usable for at least one of the provided user agents
+      #
+      # @param agents [UserAgent::Browsers::Base]
+      #     List of user agents to figure out if it could be a Pod2Watch browser on Apple Watch
+      #
+      # @return [Boolean]
+      #     true, if this is the right class for the user agent
+      def self.extend?(agents)
+        agents.first.product.start_with?(BROWSER) if agents.size.positive?
```

Comment:
Curious why here we only check the first agent instead of looping through the agents? This contradicts to the YARD also.

Reply:
Not sure (this is how it was previously), but this does seem odd. @ziggythehamster any thoughts?

Reply:
This is because `agents` is a list of tokens in the user agent string, and we specifically want to check that the first token (and no others) matches our browser.

In the case of ATC, we would match `lol/1.0 atc/1.0` because we don't care about which token contains `atc` as the product. This is mostly important for the `watchOS` part because we don't necessarily care which token it is.